.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Check out code with "sha1" commit
+2. Build
+3. run local test using "...." commands
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
```

Comment:
Actual behavior too?

Reply:
actual behavior should be in **Describe the bug** section.